### DISCUSSION TITLE
War broke out

### DISCUSSION THREAD

**Discussion by mush42** [mush42/bookworm#224] [General]
Hello Bookworm Community,

This is Musharraf, the lead developer of Bookworm.

Since April 15 2023, a military conflict has broke out where I live.

Although I managed to reach safety a few days ago, it is very difficult to lead a normal life where I currently live due to the consequences of the conflict. No electricity, no internet access etc.

Consequently, I will minimize my engagement with the project for an unknown period of time. This doesn't mean I'll stop maintaining the project, I will continue doing what I can. But don't expect me to actively engage.

As soon as things are back to normal, I'll back to development with full speed.

Best
Musharraf

**Comment by cary-rowen** [comment]
Hello @mush42 

Please take good care of yourself and stay safe. I hope that this conflict will come to an end soon, and that you'll be able to return to a peaceful and stable life.
Remember that I'm here for you, and that you can always reach out to me if you need someone to talk to or if there's anything I can do to support you. My thoughts and best wishes are with you.

Best
Cary
